README.md

```diff
@@ -1,2 +1,18 @@
 # autolabcli
 Codebase for backward incompatible revision of autolabcli v0.1.1.
+
+[![codecov](https://codecov.io/gh/AutolabJS/autolabcli/branch/autolabcli-ng/graph/badge.svg)](https://codecov.io/gh/AutolabJS/autolabcli)
```

Comment:
please see the [README.md of IRCLogParser](https://github.com/prasadtalasila/IRCLogParser/blob/master/README.md) project for a way to clearly indicate different code coverage reports.